docs/standard/tour.md

```diff
@@ -14,137 +14,130 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
 
-In .NET, objects that reference unmanaged resources implement the @System.IDisposable interface.  When you are done using the object, you call the object's @System.IDisposable.Dispose method, which is responsible for releasing any unmanaged resources.  .NET languages provide a convenient `using` syntax for such objects, as in the following example:
+In .NET, objects that reference unmanaged resources implement the <xref:System.IDisposable> interface. When you're done using the object, you call the object's <xref:System.IDisposable.Dispose> method, which is responsible for releasing any unmanaged resources. .NET languages provide a convenient `using` syntax for such objects, as shown in the following example:
 
 [!code-csharp[UnmanagedResources](../../samples/csharp/snippets/tour/UnmanagedResources.csx#L1-L6)]
 
-Once the `using` block completes, the .NET runtime will automatically call the `stream` object's @System.IDisposable.Dispose method, which releases the file handle.  The runtime will also do this if an exception causes control to leave the block.
+Once the `using` block completes, the .NET runtime automatically calls the `stream` object's <xref:System.IDisposable.Dispose> method, which releases the file handle. The runtime also does this if an exception causes control to leave the block.
 
-For more details, check out the following pages:
+For more details, check out the following topics:
 
 * For C#, [using Statement](../csharp/language-reference/keywords/using-statement.md)
 * For F#, [Resource Management: The `use` Keyword](../fsharp/language-reference/resource-management-the-use-keyword.md)
-* For Visual Basic, [Using Statement](../visual-basic/language-reference/statements/using-statement.md)
+* For VB, [Using Statement](../visual-basic/language-reference/statements/using-statement.md)
 
 ## Type safety
 
-Objects are allocated in terms of types. The only operations allowed for a given object, and the memory it consumes, are those of its type. A `Dog` type may have `Jump` and `WagTail` methods, but not likely a `SumTotal` method. A program can only call the declared methods of a given type. All other calls will result either in a compile-time error or a run-time exception (in case of using dynamic features or `object`).
+Objects are allocated in terms of types. The only operations allowed for a given object, and the memory it consumes, are those of its type. A `Dog` type may have `Jump` and `WagTail` methods but not likely a `SumTotal` method. A program only calls the declared methods of a given type. All other calls result in either a compile-time error or a run-time exception (in case of using dynamic features or `object`).
```

Comment:
The attempt to work in memory allocation here is confusing, and makes this murky. So
1. Objects are allocated in terms of types." --> "n object is an instance of a specific type."
2. Delete "and the memory it consumes.
3. Delete likely
4. A program can also call the inherited methods of a given type. So "declared methods" --> "methods belonging to"